ksqldb-rest-model/pom.xml

```diff
@@ -35,6 +35,12 @@
             <version>${io.confluent.ksql.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>io.confluent.ksql</groupId>
+            <artifactId>ksqldb-engine-common</artifactId>
```

Comment:
Needed because `SqlTimeTypes is in this package`. It's a very lightweight package and only has one dependency on `ksqldb-common` though.